.github/workflows/main.yml

```
@@ -0,0 +1,58 @@
+name: "Spec"
+on:
+  push:
+    branches: master
+  pull_request:
```

I've noticed it's useful to include a `workflow_dispatch` trigger, to let you run the workflow manually:

```suggestion
  push:
    branches: master
  pull_request:
  workflow_dispatch:
```

(cf https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch)